collectors/python.d.plugin/turbostat/turbostat.chart.py

```diff
@@ -0,0 +1,267 @@
+# -*- coding: utf-8 -*-
+# Description: turbostat netdata python.d module
+# Author: Steven Noonan (tycho)
+
+import copy
+import os
+import struct
+import subprocess
+import time
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
```

Comment:
Please disable by default the module

`disabled_by_default` = True

This should be mentioned in the module readme, in order to enable the module user have to explicitly enable it in the `python.d.conf`

Reply:
I'm not sure where the best place to note that is.